prisma/schema.prisma

```diff
@@ -15,9 +15,10 @@ model User {
   preferredName String?
   role          Role           @default(FACULTY)
   dateModified  BigInt         @default(dbgenerated("extract(epoch from now())::bigint"))
-  ProfessorInfo ProfessorInfo?
-  Activity      Activity[]
-  Narrative     Narrative[]
+  professorInfo ProfessorInfo?
+  scoreInfo     ProfessorScore?
```

Comment:
nitpick: why not call this professorScore?

Reply:
yup I can rename it